src/server.c

```diff
@@ -1231,7 +1231,8 @@ void cronUpdateMemoryStats(void) {
                                    &server.cron_malloc_stats.allocator_resident,
                                    NULL,
                                    &server.cron_malloc_stats.allocator_muzzy,
-                                   &server.cron_malloc_stats.allocator_frag_smallbins_bytes);
+                                   &server.cron_malloc_stats.allocator_frag_smallbins_bytes,
+                                   server.lua_arena);
```

Comment:
i think that for showing used memory, we don't want to deduct the lua arena.
i.e. starting with redis 8.0, the lua memory is part of the total memory usage of redis.

p.s. that means we need to sum it in the overheads in getMemoryOverheadData